Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Use the specified base image
+FROM nvidia/cuda:12.0.0-devel-ubuntu22.04
+
+# Update and install dependencies
+RUN apt-get update && apt-get install -y \
+    cmake \
```

Comment:
My two cents: although I suggested a Docker I think it would be good to implement it under a different PR because this requires some more testing. The first example - taking cmake from Ubuntu 20.x repo results in an old version causing compilation problems.